docs/src/api.rst

```diff
@@ -13,9 +13,9 @@ Base Classes
    :toctree: ./generated/api/base/
    :template: class.rst
 
-   base_variable_importance.BaseVariableImportance
```

Comment:
I put the full name because there are not defined in the init at the root of the library.

You should play with the parameter of autosummary.
https://www.sphinx-doc.org/en/master/usage/extensions/autosummary.html